lib/dynflow/execution_plan/hooks.rb

```diff
@@ -21,7 +21,7 @@ def initialize(hooks = {})
         # @param class_name [Class] class of the hook to be run
         # @param on [Symbol, Array<Symbol>] when should the hook be run, one of {HOOK_KINDS}
         # @return [void]
-        def use(class_name, on: HOOK_KINDS)
+        def use(class_name, on: ExecutionPlan.states)
```

Comment:
Why we did add `:success`, `:failure` before and now we do not need to? :thinking:

Reply:
I wanted to make this run the hook on all state transitions. What I overlooked was that `:success` is a shorthand for `stopped-success` state-result pair, which meant that the hook was being run twice if the plan stopped successfully.